### PR TITLE
Move Toleranced to JITX

### PR DESCRIPTION
**Pull request by ariel-hi**
This PR moves Toleranced to `jitx` package.

Updates:
- Removed `ocdb/utils/tolerance`
- Renamed min-value and max-value to `min` and `max`
- Removed imports to ocdb/utils/tolerance, and ensured that those files also import jitx

[TODO] Grab new `jitx-client` once merged, and double check changes.

### CODE REVIEW

File: components/texas-instruments/SN65HVD1786.stanza [hunk 1]
```diff
@@ -53,7 +52,7 @@ public pcb-component component :
   manufacturer = "Texas Instruments"
   name = "TI:SN65HVD1786D"
   reference-prefix = "U"
-  mpn = "SN65HVD1786DR‎"
+  mpn = "SN65HVD1786DR"
```
Comment (ariel-hi) [inline components/texas-instruments/SN65HVD1786.stanza:55]: Deleted invisible character:
Screen Shot 2022-09-19 at 12 11 29 PM

**Review by bhusang — APPROVED** [review]
Looks like you excised all mentions of toleranced from ocdb, Need to release the new version of ocdb in lock step with the updated jitx-client? And we have to adjust all userland code and example code in jitx-client itself.

**Comment by bhusang** [comment]
@ariel-hi I think that you need to make sure that the CI will use the updated version of jitx-client as well?

**Comment by ariel-hi** [comment]
Thanks for the review @bhusang 
The jitx-client PR is undergoing final edits to resolve naming collisions. Once a release with new Toleranced deployed, I will updated `.github/jitx-client/Dockerfile` to link OCDB to that release.

**Comment by ariel-hi** [comment]
I have updated `jitx-client` code and `jitpcb-by-example` documentation referencing Toleranced. There will be a follow-up PR for documentation to shuffle files around.

**Pull request merged**